contracts/rate_oracles/BaseRateOracle.sol

```diff
@@ -280,10 +266,10 @@ abstract contract BaseRateOracle is IRateOracle, Ownable {
             // more generally, what should our terminology be to distinguish cases where we represetn a 5% APY as = 1.05 vs. 0.05? We should pick a clear terminology and be use it throughout our descriptions / Hungarian notation / user defined types.
 
             if (atOrAfter.observedValue > beforeOrAt.observedValue) {
-                uint256 rateFromBeforeOrAtToAtOrAfterRay = WadRayMath.rayDiv(
-                    atOrAfter.observedValue,
-                    beforeOrAt.observedValue
-                ) - WadRayMath.RAY;
+                uint256 rateFromBeforeOrAtToAtOrAfterRay = getGrowthBetween(
```

Comment:
lol, with a fresh look this variable name is so confusing, makes it easy to misunderstand the

Reply:
hmm, I can't think of anything better tbh. maybe the function name helps with readability